components/feature/download/README.md

```diff
@@ -0,0 +1,19 @@
+# [Android Components](../../../README.md) > Feature > Download
+
+Feature implementation for apps that want to use Android downloads manager.
+
+## Usage
```

Comment:
Let's try to add some basic docs with new components: Let's describe what this actually does (forward downloads requests of the currently selected session to Android's Download Manager) and how to use it.